components/internal/Table.tsx

```diff
@@ -87,6 +88,7 @@ export default function Table(props: Props) {
                 key={row.id}
                 onClick={onRowClick ? () => onRowClick(row.original) : undefined}
                 cursor={onRowClick ? 'pointer' : undefined}
+                _hover={{ background: 'background.interactive' }}
```

Comment:
Looks like we only need hover colour change when clicking on rows does something, so we can do:
```suggestion
                _hover={onRowClick && { background: 'background.interactive' }}
```